eatngo-core/src/main/kotlin/com/eatngo/store/serviceImpl/StoreServiceImpl.kt

```
@@ -0,0 +1,135 @@
+            pickupAvailableForTomorrow = request.pickupAvailableForTomorrow
+        )
+
+        // 4. 저장 후 DTO 변환
+        val savedStore = storePersistence.save(updatedStore)
+        return savedStore.toDto()
+    }
+
+    override suspend fun updateStoreStatus(id: Long, status: String): StoreDto {
+        val storeStatus = try {
+            StoreEnum.StoreStatus.valueOf(status.uppercase())
+        } catch (e: IllegalArgumentException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)